docs/prerequisites.md

```diff
@@ -0,0 +1,76 @@
+# Deployment of Metal3 on vanilla K8s cluster
+To deploy metal3-components on vanilla K8s cluster, the following prerequisites
+have to be met:
+1. **Ironic should have access to layer-2 network for provisioning.**
+2. **Firewall is configured properly**
+3. **Webserver container containing  node images is running and reachable**
+4. **Ironic-bmo-configmap is populated correctly**
+
+We elaborate these points in detail here:
+1. Ironic should have access to layer-2 network for provisioning. It should be
+running on **host** networking. And on top of that the network should be
+configured so that nodes can reach the networking service for DHCP, PXE boot .
+It is also required to provide ironic instrospection with the MAC address(es) of
+ each node that ironic is provisioning so that it can pass this information to
+ networking service for DHCP and PXE boot configuration.
```

Comment:
That's not what the mac is for as far as I know. It is used when the driver is ipmi for example ( or any other driver that does not allow you to query the mac address directly) when the introspection data is returned

Reply:
Hm I misunderstood then. Your second line is not clear to me. What do you mean it is used ... when the introspection data is returned?

Reply:
yes, to know what host the introspection data is from

Reply:
ok understood. Thanks